tinyagentos/scheduler/gpu_arbiter.py

```
@@ -0,0 +1,473 @@
+                )
+            return GpuAdmission(
+                admitted=True, free_vram_mb=effective_free,
+                required_vram_mb=required_vram_mb,
+            )
+        if self._cluster_manager is not None:
+            leases = self._cluster_manager.get_leases()
+            for worker in self._cluster_manager.get_workers():
+                if worker.status != "online":
+                    continue
+                worker_leases = sum(
+                    l.required_vram_mb for l in leases
```

Ruff (0.15.20)

[error] Ambiguous variable name: `l`

(E741)